src/SprykerEco/Zed/Econda/Business/Exporter/Runner.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Exporter;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Shared\Kernel\Store;
+use SprykerEco\Zed\Econda\Business\Exporter\Exception\BatchResultException;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleFacadeInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Runner implements RunnerInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface
+     */
+    protected $exporter;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleFacadeInterface
+     */
+    protected $localeFacade;
+
+    /**
+     * @param \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleFacadeInterface $localeFacade
+     * @param \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface $exporter
+     */
+    public function __construct(
+        EcondaToLocaleFacadeInterface $localeFacade,
+        ExporterInterface $exporter
+    ) {
+        $this->localeFacade = $localeFacade;
+        $this->exporter = $exporter;
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return array
+     */
+    public function runExport(OutputInterface $output): array
+    {
+        $storeCollection = Store::getInstance()->getAllowedStores();
+
+        $results = [];
+
+        foreach ($storeCollection as $storeName) {
+            $output->writeln('');
+            $output->writeln('<fg=yellow>----------------------------------------</fg=yellow>');
+            $output->writeln(sprintf(
+                '<fg=yellow>Exporting Store:</fg=yellow> <fg=white>%s</fg=white>',
+                $storeName
+            ));
+            $output->writeln('');
+
+            $localeCollection = Store::getInstance()->getLocalesPerStore($storeName);
+            foreach ($localeCollection as $localeCode) {
+                $localeTransfer = $this->localeFacade->getLocale($localeCode);
+                $results[$storeName . '@' . $localeCode] = $this->exportStoreByLocale($localeTransfer, $output);
+            }
+        }
+
+        return $results;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\LocaleTransfer $locale
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return array
+     */
+    protected function exportStoreByLocale(LocaleTransfer $locale, OutputInterface $output): array
+    {
+        $results = [];
+        $types = $this->getEnabledExports();
```

Comment:
`$types = getEnabledExporters() = getCollectorPlugins() = EcondaPluginInterface = ProductsPlugin = CategoriesPlugin`

It's very difficult to follow this. Everything is named so differently.

$types => $availableExporterTypes
DependencyProvider::PRODUCTS => DependencyProvider::EXPORTER_TYPE_PRODUCT
getEnabledExporters() => getAvailableExporterTypes()
getCollectorPlugins() => getExporterPlugins()
EcondaPluginInterface => ExporterPluginInterface
ProductsPlugin => ProductExporterPluginInterface
CategoryPlugin => CategoryExporterPluginInterface

This will end up with the chain:
`$availableExporterTypes = getAvailableExporterTypes() = getExporterPlugins() => ExporterPluginInterface => ProductExporterPlugin`